.iex.exs

```diff
@@ -0,0 +1,3 @@
+alias AeMdw.Db.{Model, Name, Util}
+require Model
+require Ex2ms
```

Comment:
Perhaps we can get rid of the abbreviated alias notation, credo won't complain about it because it won't scan this file, but the full notation is always more useful because it can be searched for easily.

Reply:
Done